packages/integration-sdk-cli/src/commands/options.ts

```diff
@@ -69,18 +66,6 @@ export function addSyncOptionsToCommand(command: Command): Command {
       'integration-managed',
     )
     .option('--scope <anystring>', 'specify synchronization job scope value')
-    .option(
-      '-u, --upload-batch-size <number>',
-      'specify number of entities and relationships per upload batch',
-      (value, _previous: Number) => Number(value),
-      DEFAULT_UPLOAD_BATCH_SIZE,
-    )
-    .option(
-      '-ur, --upload-relationship-batch-size <number>',
-      'specify number of relationships per upload batch, overrides --upload-batch-size',
-      (value, _previous: Number) => Number(value),
-      DEFAULT_UPLOAD_BATCH_SIZE,
-    )
```

Comment:
If someone was previously using the `upload-batch-size` parameter and we remove it, will it break their script?
I understand it isn't actually being used, but I want to know if this will create a hard failure in their script.

Reply:
Yes, this is a breaking change. I assume we have a mechanism to mark it as such - PR label? I'll add an update to the changelog.